blocks/grid/grid.editor.js

```diff
@@ -0,0 +1,215 @@
+export default function config() {
+  return {
+    inline: {
+      component: 'InlineEditGridComponent',
+      inputs: [],
+    },
+    attributes: {
+      grid: {
+        'template-rows': {
+          type: 'text',
+          label: 'Row',
+          helpText: 'The row number.',
+          value: '1fr',
+        },
+        'template-columns': {
+          type: 'text',
+          label: 'Columns',
+          helpText: 'The column number.',
+          value: '1fr 1fr',
+        },
+        gap: {
+          type: 'text',
+          label: 'Gap',
+          helpText: 'The gap between the grid items.',
+          value: '20px',
+        },
+      },
+      data: {
+        level: {
```

Comment:
If nested grids is not supported level is not needed anymore it can be removed from here and from the component.